### PR TITLE
Added instructions for installing EPEL and RPM Fusion.

### PR DESCRIPTION
**Pull request by rl2939**
## What does this pull request do?

This pull request adds instructions for adding the EPEL and RPM Fusion software repositories, with GPG key verification.

## Why?

This pull request aims to make the process of installing several software repositories more clear for the end user.

The current documentation does not explain how to install EPEL in a clear manner (see https://github.com/AlmaLinux/wiki/issues/232). This pull request changes the instructions to the official quickstart guide on the Fedora project website.

In addition, there are no instructions on how to install RPMfusion (which provides additional packages) anywhere on the website. The RPMfusion website does have directions on how to install their repositories, but does not do it in a way where the user can verify the GPG signing key in an easy manner. This pull request adds instructions on how to install RPM Fusion as well as verifying the signing keys.

Also included is an guide on how to install both EPEL and RPM Fusion. This is at the bottom of the documentation section so that new users can find it.

fixes https://github.com/AlmaLinux/wiki/issues/232

### CODE REVIEW

**Comment by bennyvasquez** [comment]
@xsub @rl2939 it's not clear to me, so I wanted to check. Was this content in this PR included as part of #265?

**Comment by xsub** [comment]
Hi @rl2939!  

I initially cherry-picked your commit and then attempted to enhance it, which unfortunately led to some confusion with versions during the merge. That's why my commit is missing it – my apologies 😅

Regarding the enhancements:

- I completely get your idea to rename "system series" to "post installation." However, I've decided to skip that for now. This is partly because the [After-Installation Guide](https://wiki.almalinux.org/documentation/after-installation-guide.html) by @sboldyreva was recently added, which includes "enable-necessary-repositories." Your content could easily link to this guide...

- Considering the above, I was thinking maybe we could move your instructions to a dedicated "System Series" how-to?


I’d really value your thoughts on this, @rl2939, and @bennyvasquez, I’d also appreciate your input. 
Your insights would be very helpful in defining our approach 😊

**Comment by xsub** [comment]
@rl2939, I initially cherry-picked your commit and then attempted to enhance it, which unfortunately led to some confusion with versions during the merge. That's why my commit is missing it – my apologies 😅

Regarding the enhancements:

- I completely get your idea to rename "system series" to "post installation." However, I've decided to skip that for now. This is partly because the [After-Installation Guide](https://wiki.almalinux.org/documentation/after-installation-guide.html) by @sboldyreva was recently added, which includes "enable-necessary-repositories." Your content could easily link to this guide...

- Considering the above, I was thinking maybe we could move your instructions to a dedicated "System Series" how-to?

Let me know what you think, @rl2939 & @bennyvasquez. Your insights would be very helpful in defining our approach for the future.

File: docs/repos/Extras.md [hunk 1]
```diff
@@ -10,7 +10,11 @@ EPEL packages are usually based on their Fedora counterparts and will never conf
 
 | Repository | How to Enable |
 | --- | --- |
-| EPEL | # dnf install epel-release |
+| EPEL | See the [EPEL Quickstart Guide](https://docs.fedoraproject.org/en-US/epel/#_quickstart) |
```
Comment (bennyvasquez) [inline docs/repos/Extras.md:13]:
```suggestion
| EPEL and CRB | # dnf install epel-release |

For more detailed information see the [EPEL Quickstart Guide](https://docs.fedoraproject.org/en-US/epel/).
```

File: docs/repos/Extras.md [hunk 2]
```diff
@@ -19,3 +23,10 @@ EPEL packages are usually based on their Fedora counterparts and will never conf
 | Repository | How to Enable |
 | --- | --- |
 | ELRepo | # dnf install elrepo-release |
+
+## RPM Fusion
+[RPM Fusion](https://rpmfusion.org/) provides software that RHEL and other Enterprise Linux distributions do not want to ship (like software codecs).
```
Comment (bennyvasquez) [inline docs/repos/Extras.md:28]:
```suggestion
[RPM Fusion](https://rpmfusion.org/) provides software that RHEL and other Enterprise Linux distributions do not to ship (like software codecs).
```

File: docs/documentation/epel-and-rpmfusion.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Installing EPEL and RPM Fusion"
+---
+
+# Installing EPEL and RPM Fusion
+
+This guide describes how to install _Extra Packages for Enterprise Linux (EPEL)_ and _RPM Fusion_, two software repositories that contain software that is not included by AlmaLinux (like codecs).
+
+:::warning
+These third-party software repositories are not supported by AlmaLinux's [ELevate Project](../elevate/).
```
Comment (bennyvasquez) [inline docs/documentation/epel-and-rpmfusion.md:10]:
```suggestion
These third-party software repositories may not be supported by AlmaLinux's [ELevate Project](../elevate/).
```

**Review by bennyvasquez — COMMENTED** [review]
A few suggestions here, and I think we'll be ready to merge this in, @rl2939. Thanks for your patience!

**Comment by bennyvasquez** [comment]
@rl2939 If you don't mind, can you also rebase to pull in the updated config.js and add your guides into the updated one?

**Comment by rl2939** [comment]
@bennyvasquez I updated the config.js to match the current one. Will this work?

**Review by bennyvasquez — APPROVED** [review]

**Comment by bennyvasquez** [comment]
Thanks for your patience and work here, @rl2939! So glad to get this merged in.

**Pull request merged**